deployment/base/scripts/init_data/01.Partner.template.ini

```diff
@@ -196,13 +196,22 @@ status=1
 
 [provisioner]
 id=-22
-prefix=-23
+prefix=-22
```

Comment:
I'm not sure about updating an existing partner, what is the effect of this update?